nodes/ui_base.html

```diff
@@ -1874,11 +1874,12 @@
                         }
                     }
                     // buttons
+                    var nodeId = item.node.id.split('.');
                     var buttonGroup = $('<div>',{class:"nr-db-sb-list-header-button-group"}).appendTo(titleRow);
                     if (item.type === 'ui_tab') {
-                        var addGroupButton = $('<a href="#" class="editor-button editor-button-small nr-db-sb-list-header-button"><i class="fa fa-plus"></i> '+c_("layout.group")+'</a>').appendTo(buttonGroup);
+                        var addGroupButton = $('<a href="#" class="tab-add-group-button-' + nodeId[0] + '-' + nodeId[1] +' editor-button editor-button-small nr-db-sb-list-header-button" ><i class="fa fa-plus"></i> '+c_("layout.group")+'</a>').appendTo(buttonGroup);
                     }
-                    var editButton = $('<a href="#" class="editor-button editor-button-small nr-db-sb-list-header-button"><i class="fa fa-pencil"></i> '+c_("layout.edit")+'</a>').appendTo(buttonGroup);
+                    var editButton = $('<a href="#" node-id"'+item.node.id+'" class="tab-edit-button-'+nodeId[0]+'-'+nodeId[1]+' editor-button  editor-button-small nr-db-sb-list-header-button"><i class="fa fa-pencil"></i> '+c_("layout.edit")+'</a>').appendTo(buttonGroup);
```

Comment:
So here you do use node-id ? so do you need both ?
but again can we make the classes nr-db-sb-...   (short for nodered-dashboard-sidebar... etc)